app/code/core/Mage/Adminhtml/Block/Sitemap/Grid/Renderer/Time.php

```diff
@@ -19,10 +19,10 @@
  */
 
 /**
- * Sitemap grid link column renderer
+ * Sitemap grid link column rendererMage_Catalog_Block_Seo_Sitemap_Tree_Pager
```

Comment:
I don't think you meant to do this change

Reply:
Thanks